lib/teach-api.js

```diff
@@ -84,10 +84,17 @@ _.extend(TeachAPI.prototype, {
   },
   request: function(method, path) {
     var info = this.getLoginInfo();
-    var req = request(method, urlResolve(this.baseURL, path));
+    var url = urlResolve(this.baseURL, path);
+    var req = request(method, url);
 
     if (info && info.token) {
-      req.set('Authorization', 'Token ' + info.token);
+      if (url.indexOf(this.baseURL + '/') == 0) {
```

Comment:
Eh, no, the === 0 would be better. string.indexOf logic always makes me do a double look.